pkg/config/config_template.yaml

```diff
@@ -1579,11 +1579,21 @@ api_key:
   # - ns2
   # - system-ns
 
-  ## @param trace_buffer - integer - optional
-  ## @env DD_APM_TRACE_BUFFER - integer - optional
-  ## Specifies the size of trace payloads to buffer before dropping them. If unset, trace payloads are unbuffered.
-  #
-  # trace_buffer: 100
+  ## @param trace_buffer - integer - optional - default: 0
+  ## @env DD_APM_TRACE_BUFFER - integer - optional - default: 0
+  ## 
+  ## WARNING: Do not use this config. It is here for debugging and
+  ## as a temporary fix in certain load scenarios. Setting this will
+  ## result in a performance deterioration and an increase in memory
+  ## usage when the trace agent is under load. This config may be
+  ## removed in a future version.
+  ## 
+  ## Specifies the number of trace payloads to buffer after decoding.
+  ## Traces can be buffered when receiving traces faster than the
+  ## processor can process them.
+  ## 
```

Comment:
```suggestion
  ## If unset, trace payloads are unbuffered.
```

Reply:
I think it's still useful to clarify the default behavior. Readers may not understand what it means with `default: 0`

Reply:
Discussed elsewhere.